docs/customer_management/customer_portal.md

```diff
@@ -92,9 +92,9 @@ All roles and policies associated with them can be fully customized to fit your
 
 Members can be invited to the organization from:
 
-- the Back Office, go to **Customers** -> **Companies** -> **Select your company** -> **Invitations** -> **Invite member**
+- the Back Office: go to **Customers** -> **Companies** -> **Select your company** -> **Invitations** -> **Invite member**
```

Comment:
We are using comma in such sentences (there is even example below "n the Back Office, go to **Admin** (...)), so I would keep it as it was.
```suggestion
- the Back Office, go to **Customers** -> **Companies** -> **Select your company** -> **Invitations** -> **Invite member**
```

Reply:
"yes but" we have a preceding part of the sentence that goes "Members can be invited to the organization from..." and "the Back Office:" with a colon completes it nicely. WDYT @julitafalcondusza @juskora ?

Reply:
I would update the sentence: "Members can be invited to the organization from:" -> "Members can be invited to the organization in the following ways:". Then, "**in** the Back Office, (...)".